File: 7-수학/11005.py
import sys
import math
sys.stdin = open("ex.txt","r")
[a, b] = map(int,sys.stdin.readline().rstrip().split(' '))

def solution(x):
    if (x < 10):
        return x
    return chr(x + 55)

answer = ''
if (a == 0):
    answer = 0
else:
    while(a > 0):
        tmp = solution(a % b)
        answer = str(tmp) + answer
        a = math.floor(a / b)

print(answer)

File: 7-수학/11005.js
const fs = require('fs')
const input = fs.readFileSync('ex.txt').toString().trim().split(' ')

arr = input.map(v => Number(v))
let a = input[0]
let b = input[1]
let answer = []
while (a / b !== 0) {
    answer.push(a % b)
    a = Math.floor(a / b)
}
answer = answer.reverse()

for (let i = 0; i < answer.length; i++) {
    answer[i] >= 10 && answer[i] <= 35
        ? (answer[i] = String.fromCharCode(answer[i] + 55))
        : 0
}

console.log(answer.join(''))
